src/css/structure/modules/infragistics.ui.grid.css

```diff
@@ -2124,4 +2124,13 @@ margin:1em 20%;
 
 .ui-iggrid-featurechooser-list-submenu li.ui-state-hover {
     background:#eee;
-}
\ No newline at end of file
+}
+
+/* BUG 220509 11.12.2017 Marin Popov */
+.ui-widget .ui-igedit.ui-state-disabled {
```

Comment:
This applies to both rules - scope them to `.ui-iggrid` or something similar, otherwise they apply outside the grid which is not intended.